lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -132,7 +194,8 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
           finished?: boolean,
           window_duration: non_neg_integer,
           discontinuities_counter: non_neg_integer,
-          next_segment_id: non_neg_integer()
+          next_segment_id: non_neg_integer(),
+          segment_sequencer: {non_neg_integer(), non_neg_integer}
```

Comment:
would be helpful if you could tag what given element in the tuple is such as `{msn :: non_neg_integer(), part_sn :: non_neg_integer()}` or whatever given numbers mean.